### PR TITLE
update cli help

### PR DESCRIPTION
**Pull request by cs01**
Add an additional note at the bottom directing users to the docs for more configuration options.

Note: my text editor is configured to autoformat with [black](https://github.com/ambv/black), so I saved the first commit with formatting changes only. If you don't want these formatting changes I can back them out. See 245ca15 for the cli change.

### CODE REVIEW

File: pdoc/cli.py [hunk 1]
```diff
@@ -16,127 +16,115 @@
 import pdoc
 
 parser = argparse.ArgumentParser(
-    description="Automatically generate API docs for Python modules.",
+    description=(
+        "Automatically generate API docs for Python modules. "
+        "Documentation is available at https://pdoc3.github.io/pdoc/doc/pdoc/."
```
Comment (kernc) [inline pdoc/cli.py]: <[link in angle brackets](https://www.w3.org/Addressing/URL/5.1_Wrappers.html)>

A shorter URL https://pdoc3.github.io/pdoc/doc/ should work.

Comment (kernc) [reply pdoc/cli.py]: I'd move this statement into the epilog and have just the one.

**Comment by kernc** [comment]
Thanks!

**Comment by cs01** [comment]
flake8 is cool too for sure. I am lazy and like the tool to do the reformatting for me, not just report errors.

Thanks for the merge!

**Pull request merged**